targetFunctions/common/reflectivityCalculations/abeles/findkn.m

```diff
@@ -1,30 +1,6 @@
-function kn = findkn(k0,sld)
+function kn = findkn(k0, sld)
```

Comment:
Will it be useful to keep some comments?

Reply:
Hmm, I was wondering about that. I'll take another look and see.